acceptable/mocks.py

```diff
@@ -0,0 +1,461 @@
+# Copyright 2019 Canonical Ltd.  This software is licensed under the
+# GNU Lesser General Public License version 3 (see the file LICENSE).
+from future import standard_library
+
+standard_library.install_aliases()
+
+from collections import namedtuple
+import copy
+from json import dumps as json_dumps
+from json import loads as json_loads
+from urllib.parse import urljoin
+import responses
+from acceptable._validation import validate
+import re
+from requests.utils import CaseInsensitiveDict
+
+
+class Attrs(object):
+    def __init__(self, attrs):
+        self._attrs = attrs
+    
+    def __dir__(self):
+        return list(self._attrs)
+    
+    def __getattr__(self, name):
+        try:
+            return self._attrs[name]
+        except KeyError:
+            raise NameError(name)
+
+    def __iter__(self):
+        return iter(self._attrs.items())
+
+
+class ResponsesManager(object):
```

Comment:
Docstring explaining why we need a responses manager?

Reply:
Good idea